MonoGame.Framework/Input/GamePadDPad.cs

```diff
@@ -74,7 +74,7 @@ public GamePadDPad(ButtonState upValue, ButtonState downValue, ButtonState leftV
             Right = rightValue;
         }
 
-        internal GamePadDPad(params Buttons[] buttons)
+        internal GamePadDPad(Buttons[] buttons)
```

Comment:
@06needhamt - I think I misled you a bit in https://github.com/mono/MonoGame/issues/4910.  We need to remove the array completely...  there is no need for any array here.  The function should be `internal GamePadDPad(Buttons buttons)`.

This of course means the `foreach` loop is removed with it just testing the single `buttons` type.